src/ctia/schemas/graphql/helpers.clj

```diff
@@ -88,10 +106,12 @@
                                    values
                                    default-type-registry))
   ([^String enum-name ^String description values registry]
-   (or (get @registry enum-name)
-       (let [builder (-> (GraphQLEnumType/newEnum)
+    (get-or-update-type-registry
```

Comment:
Please temporarily permit the funny indentation, a forthcoming commit with wrap this entire return value in a function and restore some order.